iree_tests/configs/config_onnx_cpu_llvm_sync.json

```diff
@@ -1,15 +1,14 @@
 {
   "config_name": "cpu_llvm_sync",
-  "iree_compile_flags" : [
+  "iree_compile_flags": [
     "--iree-hal-target-backends=llvm-cpu"
   ],
   "iree_run_module_flags": [
     "--device=local-sync"
   ],
   "skip_compile_tests": [
-    // TODO(#16666): Fix these tests hanging at runtime (miscompiles?)
```

Comment:
Could try to keep comments in the files but that's hard when round tripping from JSON/JSONC. I figure we'll update generated tests with automation (no comments, just lists of test cases) and authored/curated tests manually (comments allowed)